linkerd/app/inbound/src/http/router.rs

```diff
@@ -245,7 +245,7 @@ impl<C> Inbound<C> {
                 .push(svc::NewOneshotRoute::layer_via(|t: &policy::Permitted<T>| {
                     LogicalPerRequest::from(t)
                 }))
-                .push(self::metrics::layer(rt.metrics.request_count.clone()))
+                .push(self::metrics::layer(&rt.metrics))
```

Comment:
this is the point: pass in `rt.metrics` so that we don't have to keep adding a new parameter each time that we add a new layer in the future.